SPECS/python-gevent/python-gevent.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Coroutine-based network library
 Name:           python-gevent
 Version:        1.3.6
-Release:        8%{?dist}
+Release:        9%{?dist}
```

Comment:
Small request: could you add `BuildRequires: python3-pip` to the dependencies for the `if %{with_check}` section below? It looks like the tests were always failing to run, because we `pip` was not there to install `nose`.

Reply:
Added. :)